src/drafts/CSSComponent/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import merge from 'classnames'
+import classNames from './component.module.css'
+
+export const Component: React.FC<React.HTMLProps<HTMLDivElement>> = ({className, ...props}) => {
```

Comment:
What purpose does this component serve?

Reply:
No purpose to the user, I added this as a temporary component to show in storybook and to build with rollup

Reply:
Isn't that what Button2 was for?

Reply:
Yep, that PR became too messy to maintain, so I wanted to set up a generic dev environment, before getting into Button specifics in a new PR